packages/@ourworldindata/core-table/src/CoreTable.ts

```diff
@@ -215,6 +215,19 @@ export class CoreTable<
                 inputColumnsToParsedColumnStore
             )
 
+        // If we ever pass Mobx observable arrays, we need to convert them to regular arrays.
+        // Otherwise, operations like `.concat()` will break in unexpected ways.
+        // See https://github.com/mobxjs/mobx/blob/mobx4and5/docs/best/pitfalls.md
+        // Also, see https://github.com/owid/owid-grapher/issues/2948 for an issue caused by this problem.
+        for (const [slug, values] of Object.entries(columnStore)) {
+            const valuesThatMayBeMobxProxy = values as CoreValueType[] & {
```

Comment:
Can you pull out the type definition? I misread this and tried to parse & { toJS?...} as JS which was very confusing :)